src/fetch/transfer_file.py

```diff
@@ -50,32 +50,6 @@ def manual_propose(dune: DuneFetcher) -> None:
     )
 
 
-def post_to_slack(
```

Comment:
This method was moved into its own file (never really belonged here). See new file `src/slack.py`